server/routerlicious/packages/services-client/src/interfaces.ts

```diff
@@ -7,3 +7,19 @@ export interface IAlfredTenant {
     id: string;
     key: string;
 }
+
+export interface IDocumentSession {
+    documentId: string;
```

Comment:
Can we change this to `id`? Or is the plan to alter the driver to read `documentId` from response body instead of `id` like it would in #9411? If that other PR merges first this will need to change to `id`, otherwise the other PR needs to change to `documentId`

FYI @nedalhy potential conflicts etc.